src/sgame.c

```diff
@@ -70,7 +70,7 @@ static MenuResult StartGame(void)
             {
                 screenchanged = true;
 
-                SDLKey key = ev.key.keysym.sym;
```

Comment:
Please introduce a global typedef (in `main.h` preferably) for `CompatSDLKey` which is ifdef-d by SDL version. I'd like to keep these parts of the code independent from SDL version, and pull these changes to master.

Reply:
Sure.